.gitignore

```
@@ -0,0 +1,7 @@
+
+modules/special-rules/.DS_Store
+modules/special-rules/.DS_Store
+modules/.DS_Store
+modules/special-rules/.DS_Store
+modules/special-rules/.DS_Store
+modules/.DS_Store
```

Or rather, this suggestion will replace all the lines, so this would be the one to commit.
```suggestion
.DS_Store
```